sdk/test/UnitTests/AWSXRayRecorderTests.cs

```diff
@@ -1052,6 +1052,35 @@ public void TestDefaultStreamingStrategyWithNegativeValue()
             Assert.ThrowsException<ArgumentException>(() => new AWSXRayRecorderBuilder().WithStreamingStrategy(new DefaultStreamingStrategy(-100)));
         }
 
+        [TestMethod]
+        public void TestBeginSubsegmentWithCustomTime()

```

Comment:
Sanity check : unit test that checks begin/end subsegment if no value is passed? The assertion should be on the current time.